src/bot/con.py

```diff
@@ -0,0 +1,26 @@
+{'game_short_name': None, 
+'chat_instance': '-2248192099587011709', 
+'id': '2469572960604862310', 
+'from_user': {'id': 574992261, 'is_bot': False, 'first_name': 'Raman', 'username': None, 'last_name': None, 
```

Comment:
yeah here also id is hardcoded!

Reply:
That's just an example response 
I use that as reference